lib/fluent/plugin/out_mongo.rb

```diff
@@ -3,12 +3,13 @@
 require 'fluent/plugin/output'
 require 'fluent/plugin/mongo_auth'
 require 'fluent/plugin/logger_support'
+require 'fluent/plugin/filter'
```

Comment:
Is this `require` is needed?

Reply:
You are right. I have removed the unused require